docs/source/guides/hyperparameter_optimization.md

```diff
@@ -0,0 +1,48 @@
+# Hyperparameter Optimization
```

Comment:
Maybe you could add a link to the wandb documentation on HPO somewhere on this page for further reference